www/src/layouts/index.js

```diff
@@ -13,13 +13,13 @@
 
 import React, {Component} from 'react';
 import docsearch from 'docsearch.js';
-import Flex from 'components/Flex';
-import Footer from './components/Footer';
-import Header from './components/Header';
+import Flex from '../components/Flex';
+import Footer from '../layout-components/components/Footer';
```

Comment:
You'll probably want to either combine layout-components with the rest of the components or pick a better name for it.